<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlgorithmGui</class>
 <widget class="QWidget" name="AlgorithmGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flika - Myoquant Plugin</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="analyze_tab_widget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Set Thresholds</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_18">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Select Window</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="7" column="0">
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="0" column="0">
            <widget class="QPushButton" name="fill_boundaries_button">
             <property name="text">
              <string>Fill Boundaries</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <widget class="QGroupBox" name="circle_clusters_groupbox">
           <property name="title">
            <string>Adjust thresholds while viewing the 'binary markers' window.</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="1" column="0">
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="2" column="1">
               <layout class="QGridLayout" name="gridLayout_16"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Threshold 2</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Threshold 1</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <layout class="QGridLayout" name="gridLayout_9"/>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Set regions you are certain are cells to black. Set regions you are certain are boundaries to white.</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="8" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Set Regression Parameters</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_17"/>
         </item>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Click on segments to classify them as fibers or false positives</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="filter_clusters_layout">
                <item>
                 <widget class="QLabel" name="model_params_label">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QGridLayout" name="gridLayout_14">
           <item row="1" column="0">
            <widget class="QPushButton" name="SVM_button">
             <property name="text">
              <string>Run SVM Classification training on labeled image</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="SVM_saved_button">
             <property name="text">
              <string>Run SVM Classification on saved training data</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="analyze_tab">
        <attribute name="title">
         <string>Analyze</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>501</width>
           <height>291</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="micronsPixelLabel">
            <property name="text">
             <string>microns/pixel</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="microns_per_pixel_SpinBox">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>0.001000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0" rowspan="2" colspan="3">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Classification Validation</string>
            </property>
            <widget class="QWidget" name="formLayoutWidget">
             <property name="geometry">
              <rect>
               <x>9</x>
               <y>19</y>
               <width>481</width>
               <height>241</height>
              </rect>
             </property>
             <layout class="QFormLayout" name="validation_layout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Manually Labeled Image</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Automatically Labeled Image</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>True Positives</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>True Negatives</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>False Positives</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>False Negatives</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Precision</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Recall</string>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QLabel" name="recall_label">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QLabel" name="precision_label">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLabel" name="false_neg_label">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLabel" name="false_pos_label">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="true_neg_label">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="true_pos_label">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <layout class="QGridLayout" name="gridLayout_11"/>
              </item>
              <item row="1" column="1">
               <layout class="QGridLayout" name="gridLayout_12"/>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="f1_score">
                <property name="text">
                 <string>F1 Score</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QLabel" name="f1_score_label">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Input/Output</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_7">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>461</width>
           <height>181</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_15">
          <item row="0" column="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>138</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="save_fiber_button">
            <property name="text">
             <string>Save Fiber Data</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="mysql_export_button">
            <property name="text">
             <string>Export to MySQL</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Mouse Name:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="mousename"/>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="load_binary_button">
            <property name="text">
             <string>Load Binary Image</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
